src/vaibhav/MaxOccuranceWord.java

```diff
@@ -0,0 +1,39 @@
+package vaibhav;
+
+/*Assignment-18 : 14th Nov'2020 
+Program : 1
+max repeating word with its freq.
+input : Hi Hello Hi Techno Hello Hi
+output : Hi -> 3*/
+
+
+public class MaxOccuranceWord {
+	
+	static void occuranceWord(String str) {
+		String [] splitted=str.split(" ");
+		int max = 0;
+		String maxName = null; 
+		for (int index = 0; index < splitted.length;index++) {
+			int count = 0;
+			String word = splitted[index];	
+			if (str.contains(word)) {
+				for ( int innerIndex = 0 ; innerIndex < splitted.length ; innerIndex++) {
```

Comment:
Here you are putting one word from array to the variable called "word", & then you are checking that variable value into array, so here think logically, it will always gives you true right?
storing in one variable and checking same variable in that array makes no sense here.